HoneyBadgers.Logic/Searcher.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Runtime.InteropServices.ComTypes;
+using System.Security.Cryptography.X509Certificates;
+
+namespace HoneyBadgers.Logic
+{
+    public static class Searcher
+    {
+        public static Dictionary<Movie,int> FindByName(Data db, string searchInput)
+        {
+            var inputParts = searchInput.Split(" "); 
```

Comment:
czyli sprawdzasz czy film zawiera, którekolwiek słowo? Czyli jak wprowadzę "Ciemny las" to znajdzie mi filmy na przykład "100-milowy las" i "Ciemny dół", nie powinniśmy szukać dokładnej wartości?
Chyba że searchMatched to jest zmienna, która mówi z jaką dokładnością dany film pasuje? To świetny pomysł!! i jesli tak to możesz ją nazwać precision lub accuracy 😃